Text/Wording Improvements (#11)
1010todd/Mega-Freight: data/mships.txt

	explode "tiny explosion" 10
	explode "small explosion" 15
	"final explode" "final explosion small"

> 	description "A tugboat from early days of space trading, which was made to transport cargo containers between ground and space. Designed when repulsor engines were still in their infancy, many ships from the previous era still lack one or were not designed to enter the atmosphere at all. With more freighters equipped with repulsor engines now, the roles for these tugboats changed to assisting these large and cumbersome ships to land, not unlike their sea-bound ancestor."

Reviewer: ```suggestion
	description "A tugboat from early days of space trading made to transport cargo containers between ground and space. Designed when repulsor engines were still in their infancy, many ships from the previous era still lack one or were not designed to enter the atmosphere at all. In modern times where all freighters come equipped with repulsor engines, the roles for these tugboats changed to assisting these large and cumbersome ships to land, not unlike their sea-bound ancestor."
```
Sounds off, rephrased.

	explode "tiny explosion" 10
	explode "small explosion" 15
	"final explode" "final explosion small"

> 	description "This is a small and cheap mining vessel designed as an alternative to the Tarazed Sparrow, for pilots don't mind a little bit waiting for pay."

Reviewer: ```suggestion
	description "This is a small and cheap mining vessel designed as an alternative to the Tarazed Sparrow, for pilots don't mind a little bit of waiting for more pay."
```
That changes the meaning.

		under
	explode "tiny explosion" 10
	explode "small explosion" 15

> 	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. Also, it is often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."
> 	description `	[Carries: nothing, you shouldn't see or use this.]`

Reviewer: ```suggestion
	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. It is also often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."
	description `	[Carries: Nothing, you shouldn't see or use this.]`
```

	explode "tiny explosion" 10
	explode "small explosion" 15
	"final explode" "final explosion small"
	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. Also, it is often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."

> 	description `	[Carries: 1 Drone.]`

Reviewer: ```suggestion
	description `	[Carries: 1 Drone]`
```

	explode "tiny explosion" 10
	explode "small explosion" 15
	"final explode" "final explosion small"
	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. Also, it is often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."
	description `	This is a special variant modified by the Ankulah crew to include a towing system for towing wreckages.`

> 	description `	[Carries: 1 Small Wreckage.]`

Reviewer: ```suggestion
	description `	[Carries: 1 Small Wreckage]`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description `This is a purpose-built chemical tanker for shipping various industrial chemicals, including highly corrosive acids, with efficiency unmatched by regular freighters. Due to their specialized nature these ships are also rarer and most often found in route between industrial planets or close by. With the difficulty of efficiently maintaining transported chemicals combined with requiring special ports to unload the stored liquid, these vessels are less preferred targets for pirates. While not useful in general shipping between most worlds, these ships profit of their extreme specialization to take on fewer but much more profitable jobs which are only offered on industrialized worlds.`

Author: ```suggestion
	description `This is a purpose-built chemical tanker for shipping various industrial chemicals, including highly corrosive acids, with efficiency unmatched by regular freighters. Due to their specialized nature these ships are also rarer and most often found in route between industrial planets or close by. With the difficulty of efficiently maintaining transported chemicals combined with requiring special ports to unload the stored liquid, these vessels are a less preferred target for pirates. While not useful in general shipping between most worlds, these ships profit of their extreme specialization to take on fewer but much more profitable jobs which are only offered on industrialized worlds.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description `The Lithalsa has a specialized bitumen tanker with heated tanks to maintain the cargo liquidity and prevent it from solidifying. Though only a relatively small quantity of bitumen is required for most construction, several on going projects especially on developing worlds can require significant amount of bitumen. Because the service area for these ships often lack specialized discharge facilities the Lithalsa comes with its own discharging system for unloading its cargo directly onto trucks or other transports to distribute it to the site.`

Author: ```suggestion
	description `The Lithalsa is a specialized bitumen tanker with heated tanks to maintain the cargo liquidity and prevent it from solidifying. Though only a relatively small quantity of bitumen is required for most construction, several on going projects especially on developing worlds can require significant amount of bitumen. Because the service area for these ships often lack specialized discharge facilities the Lithalsa comes with its own discharging system for unloading its cargo directly onto trucks or other transports to distribute it to the site.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description "The Bluewhale is a massive freighter for hauling large amount of cargo across the galaxy. Due to its cost these ships are often owned by freight companies rather than individuals. While its capable of carrying much more cargo than most other freighters, one of the major complaints received from captains is that it can take very long to reach the innermost container. That said it is still far faster and easier than handling of bulk cargo."

Author: ```suggestion
	description "The Bluewhale is a massive freighter for hauling large amount of cargo across the galaxy. Due to its' cost these ships are often owned by freight companies rather than individuals. While it's capable of carrying much more cargo than most other freighters, one of the major complaints received from captains is that it can take very long to reach the innermost container. That said it is still far faster and easier than handling of bulk cargo."
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion medium"

> 	description `The Startrain is a medium ranged freighter offered by Freight Galactic with a more limited cargo capacity than other cargo ships from the same shipyard. In exchange, it is much faster to unload, equating to shorter time in port and faster deliveries.`

Author: ```suggestion
	description `The Startrain is a medium ranged freighter offered by Freight Galactic with more limited cargo capacity than other cargo ships from the same shipyard. In exchange, it is much faster to unload, equating to shorter time in port and faster deliveries.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion medium"

> 	description `This is the smallest and cheapest cargo ship currently made by Freight Galactic. Though not as well armed as their typical product, it can carry decent amount of cargo for the size. The ship is also installed with equipment to independently and rapidly unload its container for faster operation at port.`

Author: ```suggestion
	description `The smallest and cheapest cargo ship currently made by Freight Galactic. Though not as well armed as their typical product, it can carry decent amount of cargo for the size. The ship is also installed with equipment to independently and rapidly unload its container for faster operation at port.`
```

	explode "medium explosion" 25
	explode "large explosion" 35
	explode "huge explosion" 20

> 	description `Behold the Espion-exclusive light mining carrier, with sizeable squadrons of drones and fighters for defending itself, or help mining and collecting materials.`

Author: ```suggestion
	description `Behold the Espion-exclusive light mining carrier with a sizable squadron of drones and fighters for defending itself, or help mining and collecting materials.`
```

	explode "medium explosion" 25
	explode "large explosion" 35
	explode "huge explosion" 20

> 	description `The Espion-exclusive mining carrier, or rather a mobile mining and refinery station to extract and process resources on the go. Due to its large cargo holds it may spend much of its time in flight to collect a full cargo of resources. Because of its size and slowness, the vessel is less than ideal for mining and collecting resources itself, but instead relys on a sizeable fleet of fighters and drones to mine and collect resources for it. To protect their miners from harm as they operate far away from the mothership, the Wombat also comes with space for 4 pocket light warships to guard the miners.`

Author: ```suggestion
	description `The Espion-exclusive mining carrier, or rather a mobile mining and refinery station to extract and process resources on the go as due to its large cargo holds it may spend much of its time in flight to collect a full cargo of resources. Because of its size and slowness, the vessel is less than ideal for mining and collecting resources itself, but instead relies on a sizable fleet of fighters and drones to mine and collect resources for it. To protect their miners from harm as they operate far away from the mothership, the Wombat also comes with space for 4 pocket light warships to guard the miners.`
```

	explode "large explosion" 60
	explode "huge explosion" 60
	"final explode" "final explosion large"

> 	description "The Sovereign is a massive carrier and worldship of mysterious origins. It is said that this ship was designed many years ago for an expedition to outside of known space. No one knows the fate of the project and only a few of the ships remain today in private hands."

Reviewer: ```suggestion
	description "The Sovereign is a massive carrier and worldship of mysterious origins. It is said that this ship was designed many years ago for an expedition to outside of known space. No one knows the fate of the project and only the ships remain today in few private hands."
```
I think that changes the meaning. It's referring to the fact that these ships are the only thing left from the project and owned by a few individuals. (Which also imply that there's only a few of these ships.)

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description `A smaller and cheaper version of the company's CK50T Stahbar, with improved anti-radar capability and a lighter armament.`

Reviewer: ```suggestion
	description `Smaller and cheaper version of the company's CK50T Stahbar, with improved anti-radar capability and lighter armament.`
```
I don't think that's needed.

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description `This is a massive freighter designed to carry as much cargo as possible, and does so, exceeding the capacity of every other ship by far. Though unlike the other ships from MarStel Industries, it is very lightly armed relative to cargo capacity, making it lack the firepower to properly protect itself, and it does not have the speed to dodge any raider. Due to such reasons, it will require decent escorts to safely travel.`
> 	description `	Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded requires ungodly amount of time, equipment, and manpower, which only gets worse when loading cargo. Any less than a well developed urban spaceport will struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space, as few planets require this much cargo in a single delivery and fewer still are capable of handling it.`

Reviewer: ```suggestion
	description `A massive freighter designed to carry as much cargo as possible, and did so exceedingly that it surpass the capacity of every other ships by far. Though unlike the other ships from MarStel Industries, it is very lightly armed relative to cargo capacity, making it lack the firepower to properly protect itself and neither does it have the speed to dodge any raider. Due to such reasons, it will require decent escort to safely travel.`
	description `	Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded requires ungodly amount of time, equipment, and manpower, which only gets worse when loading cargo. Any less than a well developed urban spaceport will struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in most of human space, as few planets require this much cargo in a single delivery and fewer capable of handling it.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description `A Solomon Voidworks protected tanker ship for shipping crude oil and other petroleum products from and across the southern part of human space. Heavily armoured and armed, the Gallant provides excellent protection against any pirates it might come across. While the difficulty of handling bulk liquid cargo deters many pirates from stealing it, reports of raid on oil tankers are not unheard of. It is suspected that they either have no other targets or they were attempts to capture the entire ship.`

Reviewer: ```suggestion
	description `A Solomons Voidworks protected tanker ship for shipping crude oil and other petroleum products from and across the Southern space. Heavily armoured and armed, the Gallant provides excellent protection against any pirates it might come across. While the difficulty of handling bulk liquid cargo deters many pirates from stealing it, reports of raid on oil tankers are not unheard of. It is suspected that they either have no other targets or they were attempts to capture the entire ship.`
```
"Solomons Voidworks" is the name of the shipyard. Southern space is what people call that area of space and this ship is likely spoken in context of human space already.

	explode "large explosion" 40
	explode "huge explosion" 10
	"final explode" "final explosion large"

> 	description `A well defended livestock transport for shipping various live exports from the southern worlds to their destinations safely. Unlike its sea-bound ancestor which can simply have an open deck for air, the cargo decks of spacefaring livestock transports require dedicated climate control systems to ensure proper air circulation and comfortable rides for the animals, as well as reduce the smells and gases for when the crew have to perform tasks within the cargo deck.` 
> 	description `	Freshly butchered meat is known to be highly preferred over frozen meat by the wealthy, creating a lucrative trade route between the South and Paradise Planets for these specialized freighters.`

Reviewer: ```suggestion
	description `A well defended livestock transport for shipping various live exports from the southern worlds to their destination safely. Unlike its sea-bound ancestor which can simply have an open deck for air, the cargo decks of a spacefaring livestock transport require dedicated climate control systems to ensure proper air circulation and comfortable ride for the animals, as well as reduce the smells and gases for when the crew have to perform tasks within the cargo deck.` 
	description `	Freshly butchered meat is known to be highly preferred over frozen meat by the wealthy, creating a lucrative trade route between the South and Paradise Planets for these specialized freighters.`
```

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion small"

> 	description `This ship is reminiscent of refuelling barges used in older days before uses of replusor engines became more widespread with freighters, when most ships would park in orbit unloading cargo via tugboats and refueled by dedicated fuel barges like the Castello.`

Reviewer: ```suggestion
	description `Reminiscent of refuelling barges used in older days before uses of replusor engines became more widespread with freighters, when most ships would park in orbit unloading cargo via tugboats and refueled by dedicated fuel barges like the Castello.`
```

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion small"

> 	description `A small, lightly armed freighter for short-ranged shipping from an earlier and more peaceful era. These are most often seen in more secure areas of space as they lack any effective defense against pirates and do not have the range to travel far beyond their home port.`

Reviewer: ```suggestion
	description `Small, lightly armed freighter for short-ranged shipping from an earlier and more peaceful era. These are most often seen in more secure areas of space as they lack any effective defense against pirates nor the range to travel far beyond their homeport.`
```
I don't like this more verbose phrasing, "nor" works fine. Also "homeport" is a word.

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion small"

> 	description `A slightly larger version of the Mudskipper, the Gudgeon is possibly one of the most efficient cargo vessels around, but also the most defenceless and slow for its size. Though few captains enjoy piloting them, they all agree it is one of the most robust and easy to maintain freighters they know of.`

Reviewer: ```suggestion
	description `Slightly larger version of the Mudskipper, the Gudgeon is possibly one of the most efficient cargo vessel around, but also the most defenceless and slow for its size. Though few captains enjoy piloting them, they all agree it is one of the most robust and easy to maintain.`
```
"freighters they know of" seems unnecessary.

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion small"

> 	description `A water tanker ship for transporting fresh water to nearby colonies or stations which are incapable of generating enough fresh water for themselves. The Minnow itself is slow and lightly armed, making it incapable of defending itself or escaping from any threat, which makes it less ideal for travelling to frontier worlds alone with the rise of piracy.`
> 	description `	Now that most colonies are developed enough to make their own water, these ships have become much rarer as the need for them decreases. Only a small number remain parked in systems that have stations for occasional resupply missions.`

Reviewer: ```suggestion
	description `A water tanker ship for transporting fresh water to nearby colonies or stations which are incapable of generating enough fresh water for themselves. The Minnow itself is slow and lightly armed making it incapable of defending itself or escaping from any threat, which makes it less ideal for travelling to frontier worlds alone with the rise of piracy.`
	description `	Now that most colonies are developed enough to supply their own water, these ships have become much rarer as the need for them decreases. Only a small number remain parked in systems that have stations for occasional resupply mission.`
```
supply->make just change the meaning, they can get their water by other ways than making. Also "making water" doesn't sound exactly right nor economical in most case.

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion small"

> 	description `A custom-built wine tanker for the Luna Winery made to transport wine in bulk and preserve it in the most perfect condition, with individual thermal control systems for each tank to keep the heat as stable as possible.`
> 	description `	Because these ships are lightly armed, they rarely wander away from the relative safety of heavily patrolled Paradise Planets.`

Reviewer: ```suggestion
	description `A custom-built wine tanker for the Luna Winery made to transport wine in bulk and preserve it in the most perfect condition, with individual thermal control system for each tank to keep the heat as stable as possible.`
	description `	Because these ships are lightly armed, they rarely wander away from the relative safety of heavily patrolled Paradise Planets.`
```

	explode "medium explosion" 20
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description `The Citrus-class was purposely designed and built as a fruit juice tanker for the Solacer Group, mainly for transporting frozen concentrated juice with minimal water content and maximum space efficiency. The concentrated juice would then be mixed with water and packaged at destionation facility to be distributed.`

Reviewer: ```suggestion
	description `The Citrus-class is a purpose built fruit juice tanker for the Solacer Group, mainly for transporting frozen concentrated juice with minimal water content for maximum space efficiency. The concentrated juice would then be mixed with water and packaged at destination facility to be distributed.`
```
Rephrased the first part. Water content is minimized for space efficiency.

	explode "medium explosion" 20
	explode "large explosion" 10
	"final explode" "final explosion medium"

> 	description `A specialized bulk-in bag-out ship designed in cooperation between Vefrol Dockyards and Pioneer Heavy Industries, made for the Core Food Corporation could do transportation of bulk sugar and in-ship bagging at destination to decrease port time during unloading, allowing overall faster turnaround. The ship lacks large cargo loading hatches to maintain a closed system free of contaminants and instead loads via special sealed conveyor belts built into the ship. Its' built-in bagging facilities utilize the latest technology for fast and highly efficient packaging operation. Bagged sugar can be unloaded at the port far faster than in bulk as well as having the benefit of being less reliant on specialized sugar unloading equipment and storage that most ports do not have.`

Reviewer: ... , made for the Core Food Corporation could do transportation ...
Doesn't sounds right to me.

> 	description `A specialized bulk-in bag-out ship designed in cooperation between Vefrol Dockyards and Pioneer Heavy Industries, made for the Core Food Corporation could do transportation of bulk sugar and in-ship bagging at destination to decrease port time during unloading, allowing overall faster turnaround. The ship lacks large cargo loading hatches to maintain a closed system free of contaminants and instead loads via special sealed conveyor belts built into the ship. Its' built-in bagging facilities utilize the latest technology for fast and highly efficient packaging operation. Bagged sugar can be unloaded at the port far faster than in bulk as well as having the benefit of being less reliant on specialized sugar unloading equipment and storage that most ports do not have.`

Author: ```suggestion
	description `A specialized bulk-in bag-out ship designed in cooperation between Vefrol Dockyards and Pioneer Heavy Industries, made so that the Core Food Corporation could do transportation of bulk sugar and in-ship bagging at destination to decrease port time during unloading, allowing overall faster turnaround. The ship lacks large cargo loading hatches to maintain a closed system free of contaminants and instead loads via special sealed conveyor belts built into the ship. Its' built-in bagging facilities utilize the latest technology for fast and highly efficient packaging operation. Bagged sugar can be unloaded at the port far faster than in bulk as well as having the benefit of being less reliant on specialized sugar unloading equipment and storage that most ports do not have.`
```

> 	description `A specialized bulk-in bag-out ship designed in cooperation between Vefrol Dockyards and Pioneer Heavy Industries, made so that the Core Food Corporation could do transportation of bulk sugar and in-ship bagging at destination to decrease port time during unloading, allowing overall faster turnaround. The ship lacks large cargo loading hatches to maintain a closed system free of contaminants and instead loads via special sealed conveyor belts built into the ship. Its' built-in bagging facilities utilize the latest technology for fast and highly efficient packaging operation. Bagged sugar can be unloaded at the port far faster than in bulk as well as having the benefit of being less reliant on specialized sugar unloading equipment and storage that most ports do not have.`

Reviewer: ```suggestion
	description `A specialized bulk-in bag-out ship designed in cooperation between Vefrol Dockyards and Pioneer Heavy Industries for the Core Food Corporation. This is to allow transportation of bulk sugar and in-ship bagging at destination to decrease port time during unloading, resulting in overall faster turnaround. The ship lacks large cargo loading hatches to maintain a closed system free of contaminants and instead loads via special sealed conveyor belts built into the ship. Its' built-in bagging facilities utilize the latest technology for fast and highly efficient packaging operation. Bagged sugar can be unloaded at the port far faster than in bulk as well as having the benefit of being less reliant on specialized sugar unloading equipment and storage that most ports do not have.`
```
The ship is made for Core Food Corp.

	explode "small explosion" 30
	explode "medium explosion" 20
	"final explode" "final explosion medium"

> 	description `A fast mid-sized transport offered by Najrmalas Shipworks when they saw the few competitors in that field. The Ialmuslati, while expensive, offers more speed, as well as a business-class suite for better-paying customers.`

Reviewer: ```suggestion
	description `A fast mid-sized transport offered by Najrmalas Shipworks as they saw the few competitors in the market. The Ialmuslati, while expensive, offers more speed, as well as  a business-class suite for better-paying customers.`
```
Not 100% on the flow and phrasing but this should be better.

Author: That is better, but the space is definitely unneeded.
```suggestion
	description `A fast mid-sized transport offered by Najrmalas Shipworks as they saw the few competitors in the market. The Ialmuslati, while expensive, offers more speed, as well as a business-class suite for better-paying customers.`
```

> #req|round10(bnk)|0

Reviewer: ```suggestion
	#req|round10(bnk)|0
```

Author: What does that do?

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion medium"

> 	description `A fast and well-armed transport for hauling large amounts of people across huge distances, as a non-luxury alternative to the Star Queen.`

Reviewer: ```suggestion
	description `A fast and well-armed transport for hauling large amounts of people across long distances as a non-luxury alternative to the Star Queen.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion medium"

> 	description `A huge transport for mass transportation across vast distances. The development was partially sponsored by some of the southern worlds, to provide easy means of cheap transportation for migrating workers and low-budget tourists specifically coming to the South and the Rim.`

Reviewer: ```suggestion
	description `A huge liner for mass transportation across vast distances. The development was partially sponsored by some of the southern worlds in effort to provide cheaper transportation for migrating workers and low-budget tourists, specifically those coming to the southern area.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion medium"

> 	description `A luxury sight-seeing transport, offering larger view areas for more people, allowing transport companies to offer more exclusive seats.`

Reviewer: ```suggestion
	description `A luxury sight-seeing transport, offering larger view areas for more people. This allows transport companies to offer more exclusive seats and reap in extra profit at minimal cost.`
```

	explode "large explosion" 35
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description "A class of even larger dry bulk carriers than the Jupiter as exports and demand grew. Unfortunately, due to the lack of unloading gears, it can only operate at more specially-equipped ports. In addition to its' bulk bays it can also carry significant amount of containers, allowing captains to earn additional profit."

Reviewer: ```suggestion
	description "A class of even larger dry bulk carriers than the Jupiter as exports and demand grew. Unfortunately, due to the lack of unloading gears, it's operation is limited to fewer more well equipped ports. In addition to its' bulk bays it can also carry significant amount of containers, allowing captains to earn additional profit."
```

	description `	Another part of the ship's safety includes crews specially trained to handle radioactive cargo as well as responding to various foreseeable incidents, from collision, fire, and pirates boarding, in the most unlikely case that it occurs. In addition to that, the ship is also equipped for in-flight repairs, allowing the ship to perform rudimentary repairs on itself if damaged, though there is a small delay before the repair can begin and longer delay if the ship got disabled. Because all of these facts the ship's minimum crew is almost the same as chemical tankers of larger size and operating cost is one of the highest.`

> # DSSN-[cateprefix][catalogNum] Rynchops->KR

Reviewer: Unrelated. Also I indented it so it's inline with the name.

> # DSSN-[cateprefix][catalogNum] Rynchops->KR

Author: ```suggestion
		# DSSN-[cateprefix][catalogNum] Rynchops->KR
```

	explode "tiny explosion" 18
	explode "small explosion" 15
	"final explode" "final explosion small"

> 	description `The Espion Mining Co. compact gunship to be carried by their larger carriers for guarding the mining fleets while they are away from the mothership.`

Author: Why lowercase?

> 	description `The Espion Mining Co. compact gunship to be carried by their larger carriers for guarding the mining fleets while they are away from the mothership.`

Reviewer: iirc that's the correct form (or at least what I usually saw it in). Also looks better.

> 	description `The Espion Mining Co. compact gunship to be carried by their larger carriers for guarding the mining fleets while they are away from the mothership.`

Author: I'm pretty sure you're wrong (https://www.collinsdictionary.com/us/dictionary/english/co#:~:text=Co.%20is%20used%20as%20an,the%20name%20of%20an%20organization.&text=...the%20Blue%20Star%20Amusement%20Co.) and I disagree that it looks better, but your plugin.

	explode "large explosion" 50
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description "With the recent development of new Unfettered ships and the deployment of the Pond Strider and Fleas to counter them, the Hai soon developed a new heavily armed heavy freighter that is also capable of carrying drones to defend itself when flying past risky areas."

Reviewer: ```suggestion
	description "With the recent development of new Unfettered ships and the deployment of Pond Striders and Fleas to counter them, the Hai soon developed a new heavily armed heavy freighter that is also capable of carrying drones to defend itself when flying past risky areas."
```
Feels like there's to many `the` there.

	explode "large explosion" 30
	explode "huge explosion" 20
	"final explode" "final explosion large"

> 	description `A large Wanderer freighter iterated from the Deep River using the same style of cargo pod and arrangement but twice doubling the capacity, also being more well armed than Deep River to protect itself during evacuation, though it is still not really enough to operate alone.`

Reviewer: ```suggestion
	description `A large Wanderer freighter iterated from the Deep River using the same style of cargo pod and arrangement but twice doubling the capacity. It is also better armed than Deep River to protect itself during evacuations, though it is considered by many captains to still be insufficient to fly alone.`
```
Sounds off. Rather rewrite it.
